docs/developer/discounts.mdx

```diff
@@ -0,0 +1,815 @@
+---
+title: Discounts
+sidebar_label: Discounts
+---
+
+## Introduction
+
+There are two kinds of discounts in Saleor: `Sales` and `Vouchers`.
+Discounts allow for the reduction of prices for selected variants, products, collections,
+or categories by a given percentage or fixed value.
+The sale discount is automatically applied to all included products, without requiring
+any additional actions from the user. In contrast, vouchers require the user to provide
+a code during the checkout process. Both the sale and voucher discounts can be used together.
+
+### Sale
+
+As mentioned earlier, sales are automatically applied when a product is added to the checkout.
+For instance, if a product is priced at **$9** and has a **10%** discount, it will be available for **$8.1**.
+The discount will be visible on the line prices. To calculate the total applied discount,
+subtract the `line.undiscountedTotalPrice` from the `line.totalPrice`.
+To calculate the discount on one unit, subtract the `line.undiscountedUnitPrice` from the `line.unitPrice`.
```

Comment:
```suggestion
Sales are automatically applied when a product is added to the checkout.
For instance, if a product is priced at **$9** and has a **10%** discount, it will be available for **$8.1**.
The discount will be visible on the line prices. To calculate the total applied discount,
subtract the `line.undiscountedTotalPrice` from the `line.totalPrice`.
To calculate the discount on one unit, subtract the `line.undiscountedUnitPrice` from the `line.unitPrice`.
```